### ISSUE TITLE
John Hauser's SoftFloat

### ISSUE THREAD

**Issue by no-zzz-un** [CodesInChaos/SoftFloat#1]
Is this project related to the following c software implementation of ieee 754?

http://www.jhauser.us/arithmetic/SoftFloat.html

**Comment by opless** [comment]
(From a project watcher)

Not directly. I'm porting jhauser's implementation to C# though, just the
128bit implementation, however.

If I'm successful I may post it to GitHub.

-S

On 12 May 2013, at 15:34, noun notifications@github.com wrote:

Is this project related to the following c software implementation of ieee
754?

http://www.jhauser.us/arithmetic/SoftFloat.html

—
Reply to this email directly or view it on
GitHubhttps://github.com/CodesInChaos/SoftFloat/issues/1
.

**Comment by no-zzz-un** [comment]
Are you only porting quadruple precision because performance considerations are negligible?

Sounds like an interesting project in any case - hope to see it on github eventually.

**Comment by opless** [comment]
No. The extra precision will be helpful though. Not to mention FPU
independence.

On 12 May 2013, at 18:05, noun notifications@github.com wrote:

Are you only porting quadruple precision because performance considerations
are negligible?

Sounds like an interesting project in any case - hope to see it on github
eventually.

—
Reply to this email directly or view it on
GitHubhttps://github.com/CodesInChaos/SoftFloat/issues/1#issuecomment-17781193
.

**Comment by CodesInChaos** [comment]
No, this is unrelated to John Hauser's SoftFloat. I needed high performance deterministic floats for a now abandoned project, so this project is on hold as well.

Some key differences:
1. This project is quite incomplete
2. While it uses the same format as IEEE floats it doesn't follow IEEE arithmetic exactly.
3. Main goal of this code is performance, correct rounding is less of an issue.

**Comment by Piedone** [comment]
@opless did you finish with the implementation?

**Comment by opless** [comment]
Honestly, I can't remember. I do remember moving to a segmented 64 bit solution to my issue. 

But that project is on hiatus.

**Comment by Piedone** [comment]
I see, thanks anyway.

**Comment by locus84** [comment]
> Some key differences:

dc -[0p9oi8ulkjyh7mn-=[]0p
